tests/test88.c

```diff
@@ -1,3 +1,7 @@
+#ifdef HASH_FUNCTION
+#undef HASH_FUNCTION  /* this test's output depends on the pattern of hash collisions */
```

Comment:
Ok sorry, this is not working.
(Have not tested it yesterday was a bit late here)